ibm/data_source_ibm_compute_image_template.go

```diff
@@ -63,6 +70,13 @@ func dataSourceIBMComputeImageTemplateRead(d *schema.ResourceData, meta interfac
 
 	if len(pubImageTemplates) > 0 {
 		imageTemplate := pubImageTemplates[0]
+		if most_recent {
```

Comment:
Here most_recent means we should filter based on createdate that was created recently..May be you can use same logic which we implemented for datasource of virtual instance https://github.com/IBM-Cloud/terraform-provider-ibm/blob/ad76bfde7d06789e55a4e03f5da52be702f7a247/ibm/data_source_ibm_compute_vm_instance.go#L243

Reply:
I reported a bug in softlayer-go to fix this: https://github.com/softlayer/softlayer-go/issues/131
The idea is to get the API to return the information already ordered.
How can I test this with the new version?